src/configs/__mocks__/env.ts

```diff
@@ -31,10 +31,10 @@ export type PLAN_UUID_NAMESPACE = typeof PLAN_UUID_NAMESPACE;
 export const ACTION_UUID_NAMESPACE = '1bb3c217-af1c-4749-b3df-4f6108582e3c';
 export type ACTION_UUID_NAMESPACE = typeof ACTION_UUID_NAMESPACE;
 
-export const GA_CODE = process.env.REACT_APP_GA_CODE || '';
+export const GA_CODE = '';
 export type GA_CODE = typeof GA_CODE;
 
-export const GA_ENV = process.env.REACT_APP_GA_ENV || 'dev';
+export const GA_ENV = 'test';
```

Comment:
@cKellyDesign what is the value of this variable that should be used in tests?

Reply:
Yup should be `test` to make it work in the testing environment